.github/workflows/int.yml

```
@@ -26,8 +26,8 @@ jobs:
         
     - name: Start Services
       run: |
-        docker-compose up -d --build
-        sleep 30
+        docker-compose up -d --build --env-file .env.example
```

```suggestion
        docker-compose --env-file .env.example up -d --build
```
the file s envhould come first as per docker-compose syntax